blog/_posts/2024-02-xx-scala-3.4.0-and-3.3.2-released.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.4.0 and 3.3.2 LTS released!
+---
+We are thrilled to announce the release of two awaited versions of Scala: the first one in the 3.4 minor line and the patch for the Long Term Support line.
```

Comment:
```suggestion
We are thrilled to announce the release of two versions of Scala 3: the first version in the 3.4 minor line, and a patch version in the Long Term Support line.
```

It's not idiomatic (I couldn't explain why) to just say "awaited". It would be idiomatic to say "long awaited", but saying that gives the impression that the releases are late. Maybe just omit. If you want you could say "eagerly awaited", but it seems a bit like empty marketing-speak to me.